core/models/MemberModels.py

```diff
@@ -272,5 +272,8 @@ def __str__(self):
         max_length=255,
         unique=True,
     )
+    title = models.TextField(verbose_name="Position Title",
```

Comment:
I'd make this a char field. A text field is intended to a large amount of text (paragraph +) and a char field is intended for a few words.